### PR TITLE
test behavior when token based auth is enforced

### PR DESCRIPTION
**Pull request by individual-it**

### CODE REVIEW

File: tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@webUI @insulated @disablePreviews
+Feature: enforce token auth
+
+As an administrator
+I want to be able to enforce token based auth
+So that I can improve the securety of the system by forbidding basic auth with username & password
```
Comment (phil-davis) [inline tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature]: s/securety/security

File: tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	Scenario: using OCS with oauth when token auth is enforced
+		When user "user1" requests "/ocs/v1.php/apps/files_sharing/api/v1/remote_shares" with "GET" using basic auth
+		Then the OCS status code should be "997"
+		And the HTTP status code should be "401"
+		When the user requests "/ocs/v1.php/apps/files_sharing/api/v1/remote_shares" with "GET" using oauth
+		Then the OCS status code should be "100"
+		And the HTTP status code should be "200"
+
+	@skip @issuue_core_32068
```
Comment (phil-davis) [inline tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature]: s/issuue_core_32068/issue_core_32068

**Comment by individual-it** [comment]
typos fixed, please review again

File: tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		And the user "user1" has correctly established an oauth session
+
+	Scenario: access files app with oauth when token auth is enforced
+		When user "user1" requests "/index.php/apps/files" with "GET" using basic auth
+		Then the HTTP status code should be "401"
+		When the user requests "/index.php/apps/files" with "GET" using oauth
+		Then the HTTP status code should be "200"
+
+	Scenario: using WebDAV with oauth when token auth is enforced
+		When user "user0" requests "/remote.php/webdav" with "PROPFIND" using basic auth
```
Comment (phil-davis) [inline tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature:22]: @individual-it why does this say `user0` ?
`user1` exists but `user0`  does not exist.

**Pull request merged**